crates/store/re_chunk/src/arrow.rs

```diff
@@ -0,0 +1,59 @@
+use arrow::{
+    array::{make_array, RecordBatch},
+    datatypes::{Field, Schema},
+    error::ArrowError,
+};
+
+use crate::TransportChunk;
+
+impl TransportChunk {
+    /// Create an arrow-rs [`RecordBatch`] containing the data from this [`TransportChunk`].
+    pub fn try_as_arrow_record_batch(&self) -> Result<RecordBatch, ArrowError> {
```

Comment:
Hate to be that guy but im gonna be that guy
![image](https://github.com/user-attachments/assets/54784be7-6b63-4671-bb85-1531381fc272)

```suggestion
    pub fn try_to_arrow_record_batch(&self) -> Result<RecordBatch, ArrowError> {